From e39c97f3d705919627370ccdc9e6b2b4bf50ed0a Mon Sep 17 00:00:00 2001
From: Arik Nemtsov <arik@wizery.com>
Date: Thu, 28 Jul 2011 12:02:11 +0300
Subject: [PATCH 27/32] wpa_supplicant config: add support for 11n and remove random pool

Removing the random pool makes wpa_supplicant work faster. Add 11n
support for supporting 11n in p2p GO mode.
---
 wpa_supplicant/config-android |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/wpa_supplicant/config-android b/wpa_supplicant/config-android
index 8f16e20..c71056c 100644
--- a/wpa_supplicant/config-android
+++ b/wpa_supplicant/config-android
@@ -435,7 +435,10 @@ CONFIG_DEBUG_FILE=y
 # disabled. This will save some in binary size and CPU use. However, this
 # should only be considered for builds that are known to be used on devices
 # that meet the requirements described above.
-#CONFIG_NO_RANDOM_POOL=y
+CONFIG_NO_RANDOM_POOL=y
+
+# IEEE 802.11n (High Throughput) support (mainly for AP mode)
+CONFIG_IEEE80211N=y
 
 #CONFIG_LIBNL20=y
 LIBNL=$(LIBNL_PATH)
-- 
1.7.0.4

